packages/playwright-client/types/types.d.ts

```diff
@@ -18771,76 +18840,6 @@ export interface Clock {
   setSystemTime(time: number|string|Date): Promise<void>;
 }
 
-/**
```

Comment:
Can you try to keep this part of the code at the same place in the file, so that only actually changed lines appear in the diff? I think you can do it by placing `export interface ConsoleMessage` in a different place inside overrides.d.ts.

Reply:
Hmm, I still see that more than 50 lines with ConsoleMessage definition have moved up in the file, while only `type()` definition changed.